paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletViewModel.kt

```diff
@@ -73,7 +75,7 @@ internal class WalletViewModel @Inject constructor(
             cardBrandFilter = configuration.cardBrandFilter,
             isProcessing = false,
             hasCompleted = false,
-            primaryButtonLabel = completePaymentButtonLabel(configuration.stripeIntent),
+            primaryButtonLabel = completePaymentButtonLabel(configuration.stripeIntent, linkLaunchMode),
             secondaryButtonLabel = configuration.stripeIntent.secondaryButtonLabel,
```

Comment:
We should also change this one based on `linkLaunchMode`, showing `Continue another way` if launched for payment method selection only.